games-strategy/freeciv/freeciv-2.6.0.recipe

```diff
@@ -0,0 +1,125 @@
+SUMMARY="A free Civilization-like game"
+DESCRIPTION="Freeciv is a free Civilization-like game, primarily \
+for X under Unix. It has support for multiplayer games locally or \
+over a network, and an AI which gives most people a run for their money.
+Freeciv aims to be mostly rule-compatible with Civilization II [tm], \
+published by Sid Meier and Microprose [tm]. A few rules are different \
+where we think it makes more sense, and we have lots and lots of \
+adjustable parameters to make customizing games possible.
+Freeciv has been implemented completely independently of Civilization; \
+you do not need to own Civilization to play Freeciv."
+HOMEPAGE="http://www.freeciv.org/"
+COPYRIGHT="1996-2018 The Freeciv Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://netix.dl.sourceforge.net/project/freeciv/Freeciv%202.6/$portVersion/freeciv-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7c20399198d6c7d846fed9a69b02e01134ae5340a3ae0f99d1e38063ade6c999"
+PATCHES="freeciv-$portVersion.patchset"
+ADDITIONAL_FILES="freeciv.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ -z "${ARCHITECTURES##*!$targetArchitecture*}" ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+GLOBAL_WRITABLE_FILES="
+	settings/freeciv/database.lua keep-old
+	"
+
```

Comment:
Please add the `USER_SETTINGS_FILES` variable containing files/directories that would be used by the program in the user settings folder. The format is simple:

```sh
USER_SETTINGS_FILES="
	settings/<directory> directory
	settings/<file>
"
```

If the program stores its data in `B_USER_SETTINGS_DIRECTORY/freeciv` (which _currently_ is `~/config/settings/freeciv`), then the corresponding line should be:

```
	settings/freeciv directory
```

Same goes for files, for example, if the program read its configuration from `~/config/settings/freeciv.conf`, then the corresponding entry should be:

```
	settings/freeciv.conf
```

If, the file has a template that's installed with it in `dataDir/freeciv/freeciv.example.cfg`, then the line should be:

```
	settings/freeciv.conf template $dataDir/freeciv/freeciv.example.cfg
```

Reply:
@alaviss @Begasus There currently is a problem with freeciv saving its prefs to `~/.freeciv` instead of `~/config/settings/freeciv`. Additional patches to the source code might be needed...